### DISCUSSION TITLE
New Font in Windows 10 not found

### DISCUSSION THREAD

**Discussion by tbmuc** [ImageMagick/ImageMagick#2086] [Help]
Hello I'm new to IM and have installed the 64 bit windows binary 7.0.10. However the font "Hey August" is not found and not in the IM font list (convert -list font) . How to get it into the list?

**Comment by GeeMack** [comment]
With ImageMagick you can specify a font by pointing directly to the font file itself, like this...

     ... -font "c:\windows\fonts\myfont.ttf" ...

I often just put a copy of the font file in my working folder for a particular IM project. That way I know that exact same font will be available even if I copy that folder to another machine.

**Comment by tbmuc** [comment]
Works fine. Thank you very much.
